src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsAttributeKeys.cs

```diff
@@ -47,6 +47,12 @@ internal sealed class AwsAttributeKeys
 
     internal static readonly string AttributeAWSS3Bucket = "aws.s3.bucket";
 
+    internal static readonly string AttributeAWSBedrockGuardrailId = "aws.bedrock.guardrail.id";
+    internal static readonly string AttributeAWSBedrockAgentId = "aws.bedrock.agent.id";
+    internal static readonly string AttributeAWSBedrockKnowledgeBaseId = "aws.bedrock.knowledge_base.id";
+    internal static readonly string AttributeAWSBedrockDataSourceId = "aws.bedrock.data_source.id";
+    internal static readonly string AttributeGenAiModelId = "gen_ai.request.model";
```

Comment:
To match feature parity with [Python](https://github.com/aws-observability/aws-otel-python-instrumentation/pull/209/files#diff-eb87bf439a5df054c842483f6fceed08eafc2e10b4dfbdc6215e21dc2cb79240), please move "gen_ai.request.model" declaration to AwsSpanProcessingUtil.cs file, and also add "gen_ai.system".

Reply:
Done, note that `gen_ai.system` is only defined by convention but is not used for extracting remote resources.